book/modules/module7-exercises.tex

```diff
@@ -106,8 +106,8 @@
 				\item  
 					If $\vec{n} = \mat{x\\y\\z}$ is a normal vector, then
 					\begin{align*}
-						\vec x \cdot \mat{0\\2\\2} &= 0  \quad\implies\quad 2y+2z = 0\\
-						\vec x \cdot \mat{1\\1\\1} &= 0 \quad\implies\quad x+y+z = 0.
+						\vec n \cdot \mat{0\\2\\2} &= 0  \quad\implies\quad 2y+2z = 0\\
+						\vec n \cdot \mat{1\\1\\1} &= 0 \quad\implies\quad x+y+z = 0.
```

Comment:
Put this back to `x`. Here `x` stands for `(x,y,z)` by convention.

Reply:
Should I also change the `n` in the line above to `x`?

Reply:
Ah, I didn't read the question carefully. Instead, let's keep these as `n`, but change the variables to `n_x`, `n_y`, and `n_z`.